playbooks/group_vars/all

```diff
@@ -1 +1,3 @@
 env: stage
+secure_dir: '../../configuration-secure/ansible'
+local_dir: "{{ secure_dir }}/local"
```

Comment:
Does "group_vars" get mapped into the "vars" dictionary? Because these should be defined in the vars dict, if I'm understanding the code below correctly.